src/style.css

```diff
@@ -1,3 +1,67 @@
+:root {
+  --blue-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  margin-top: 3px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  text-decoration: none;
```

Comment:
The 'text-decoration: none;' property on the '.nav__list' class might not have any effect since 'text-decoration' is typically applied to inline elements like links. Consider applying this property to '.nav__item' or '.nav__link' if the intention is to remove underlines from links.